docs/synapse/userguides/ug059_ing_sync.rst

```diff
@@ -0,0 +1,12 @@
+Ingest - Sending Data to a Remote Cortex
```

Comment:
it might be worth explaining here that only splices are synced and that there are tradeoffs for using different types of cores locally when syncing.

ram: faster locally, but recreated each run, so will send all the data upstream on each run (assuming same ingest)
sqlite: slower locally, but avoids sending data upstream when run multiple times or if data is reencountered in different ingest files.